packages/typescript/index.ts

```diff
@@ -2627,6 +2627,42 @@ export class BlobHandle {
      * User can pass `externalOwner: true` to set the blob at toURI as
      * external-owned blob.
      */
+
+    public async setModelThreshold(
+        threshold: number,
+        pos_label: string
+    ): Promise<ModelInfo> {
+        return await this.client.requestJSON<ModelInfo>({
+            method: METHOD_PUT,
+            path: '/threshold',
+            args: {
+                blob: this.getURI(),
+                threshold: threshold,
+                pos_label: pos_label,
+            },
+        });
+    }
+
+    public async delModelThreshold(): Promise<ModelInfo> {
+        return await this.client.requestJSON<ModelInfo>({
+            method: METHOD_DELETE,
+            path: '/threshold',
+            args: {
+                blob: this.uri,
```

Comment:
@TheAnkurGoswami is this not this.getUri() as well?

Reply:
please stay consistent -- use this.getURI here as well